epub_test.go

```diff
@@ -59,16 +59,16 @@ const (
 	testFontFromFileSource    = "testdata/redacted-script-regular.ttf"
 	testIdentifierTemplate    = `<dc:identifier id="pub-id">%s</dc:identifier>`
 	testImageFromFileFilename = "testfromfile.png"
-	testImageFromFileSource   = "testdata/gophercolor16x16.png"
+	testImageFromFileSource   = "testdata/sample.png"
 	testNumberFilenameStart   = "01filenametest.png"
 	testSpaceInFilename       = "filename with space.png"
-	testImageFromURLSource    = "https://golang.org/doc/gopher/gophercolor16x16.png"
+	testImageFromURLSource    = "https://file-examples.com/storage/fe072e668b64cd6ce9c9963/2017/10/file_example_PNG_1MB.png"
 	testVideoFromFileFilename = "testfromfile.mp4"
 	testVideoFromFileSource   = "testdata/sample_640x360.mp4"
-	testVideoFromURLSource    = "https://filesamples.com/samples/video/mp4/sample_640x360.mp4"
+	testVideoFromURLSource    = "https://file-examples.com/storage/fe072e668b64cd6ce9c9963/2017/04/file_example_MP4_640_3MG.mp4"
```

Comment:
Again, the old URL still works, right? Why change this?

Reply:
This wasn't working for me when I was working on the tests, that's why I changed it.

Reply:
The old video is working fine for me, but the new URL is returning a 404. I think it would be best to revert this.